community/contributing/docs/docs-style-guide.md

```diff
@@ -0,0 +1,290 @@
+
+### Avoid statements that will soon be out of date
+
+Avoid using words like "currently" and "new." A feature that is new today might not be considered new in a few months.
+
+|**Do** |**Do Not**|
+|:------|:---------|
+| In version 1.4, ... | In the current version, ...|
+| The Plugins feature provides ... | The new Plugin feature provides ...|
+
+## Documentation formatting standards
+
+### Use sentence capitalization for headings
+
+~~~~~~~~~
+# Creating a custom webhook stage
+
+## Configuring parameters for custom webhook stages
+~~~~~~~~~
+
+### Line breaks
+U#se a single newline to separate block-level content like headings, lists, images, code blocks, paragraphs, and others.
+
+### Use camel case for API objects
+
+Use the same uppercase and lowercase letters that are used in the
+actual object name when you write about API objects. The names of API objects use [Camel case](https://en.wikipedia.org/wiki/Camel_case).
+
+Don't split the API object name into separate words. For example, use CredentialsController, not Credentials Controller.
+
+Refer to API objects without saying "object," unless omitting "object" leads to an awkward construction.
+
+|**Do** |**Do Not**|
+|:------|:---------|
+| The PipelineController restarts a Stage. | The pipeline controller restarts a stage.|
+| The AmazonInfrastructureController is responsible for ... | The AmazonInfrastructureController object is responsible for ...|
+
+### Use angle brackets for placeholders
+
+Use angle brackets for placeholders. Describe what a placeholder represents.
+
+For example:
+`hal plugins repository add <unique-repo-name> --url <repo-url>`
+
+
+### Use bold for user interface elements
+
+|**Do** |**Do Not**|
+|:------|:---------|
+| Click **Fork**.| Click "Fork".|
+| Select **Other**. | Select "Other".|
+
+
+### Use italics to define or introduce new terms
+
+|**Do** |**Do Not**|
+|:------|:---------|
+|A _Stage_ is a step in a pipeline ... | A "Stage" is a step in a pipeline ...|
+
+### Use code style for filenames, directories, and paths
+
+|**Do** |**Do Not**|
+|:------|:---------|
+| Open the `rosco.yaml` file. | Open the rosco.yaml file.|
+| Go to the `/docs/tutorials` directory. | Go to the /docs/tutorials directory.|
+| Open the `/.hal/config` file. | Open the /.hal/config file.|
+
+### Lists
+Group items in a list that are related to each other. Use a numbered list for instructions that need to be completed in a specific order.
+
+ - End each item in a list with a period if one or more items in the list are complete sentences. For the sake of consistency, normally either all items or none should be complete sentences.
+ - Use the number one (1.) for ordered lists.
+ - Use (+), (* ), or (-) for unordered lists.
+ - Leave a blank line after each list.
+ - Indent nested lists with four spaces (for example, ⋅⋅⋅⋅).
+ - List items may consist of multiple paragraphs. Each subsequent paragraph in a list item must be indented by either four spaces or one tab.
```

Comment:
Uh oh. Spaces v. tabs. 🚀 

But my same question applies. For list paragraphs, I indent with spaces to line up.

Reply:
So do I. For people new to markdown, though, having a concrete guideline is perhaps better.